File: Results/DER/Translation/translation_output/no_test/codeqwen/avatar/python/java/atcoder_AGC046_A.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int X = scanner.nextInt();
        int i = 1;
        int a = 0;
        while (true) {
            a = a + X;
            if (a % 360 == 0) {
                System.out.println(i);
                break;
            }
            i += 1;
        }
    }
}
```

This Java code does the same thing as your Python code. It reads an integer from the user, adds it to a running total, and checks if the total is a multiple of 360. If it is, it prints the number of iterations it took to reach that total and breaks out of the loop. If it isn't, it increments the counter and continues the loop.

File: Results/DER/Translation/translation_output/no_test/codeqwen/avatar/java/python/atcoder_AGC046_A.py
Here is the Python equivalent of the Java code:

```python
import sys

if __name__ == "__main__":
    k = 360
    j = 1

    try:
        i = int(input())
    except ValueError:
        sys.stderr.write("Input is not an integer.\n")
        sys.exit(1)

    if 1 <= i <= 179:
        while not ((i * j) % k == 0):
            j += 1
        print(j)
    else:
        print("The input is out of range. It should be between 1 and 179.")
```

This Python code does the same thing as the Java code. It reads an integer from the user, checks if it's within the range 1 to 179, and if so, finds the smallest integer j such that i * j is divisible by 360. If the input is not an integer or is out of range, it prints an error message to stderr and exits with a non-zero status.
